app/scripts/modules/core/src/pipeline/status/ArtifactList.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { IArtifact } from 'core/domain';
+
+import './artifactList.less';
+
+export interface IArtifactListProps {
+  artifacts: IArtifact[],
+};
+
+export interface IArtifactListState {}
+
+export class ArtifactList extends React.Component<IArtifactListProps, IArtifactListState> {
+  constructor(props: IArtifactListProps) {
+    super(props);
+  }
+
+  private tooltip(artifact: IArtifact): string {
+    const tooltipEntries = [];
+    if (artifact.name) {
+      tooltipEntries.push(`Artifact: ${artifact.name}`);
+    }
+    if (artifact.type) {
+      tooltipEntries.push(`Type: ${artifact.type}`);
+    }
+    if (artifact.version) {
+      tooltipEntries.push(`Version: ${artifact.version}`);
+    }
+    if (artifact.reference) {
+      tooltipEntries.push(`Reference: ${artifact.reference}`);
+    }
+    return tooltipEntries.join('\n');
+  }
+
+  public render() {
+    const { artifacts } = this.props;
+    if (!artifacts || artifacts.length === 0) {
+      return null;
+    }
+    return (
+      <ul className="trigger-details artifacts">
+        {artifacts.map((artifact: IArtifact, i: number) => {
+          const { name, version } = artifact;
+          return (
+            <li key={`${i}-${name}`} className="break-word" title={this.tooltip(artifact)}>
+              <dl>
+                <div>
```

Comment:
I think the "type" should appear here as well, probably closer to the front

Reply:
Maybe it could even take the place of the keyword "Artifact"

Reply:
:+1: I've added a `Type: <type>` row.  Replacing the keyword "Artifact" with, for example, "docker/image" pushed the artifact's name a bit too far to the right.